survey-runner-application/survey_runner.tf

```diff
@@ -45,6 +45,36 @@ resource "aws_elastic_beanstalk_environment" "survey_runner_prime" {
     name      = "ServiceRole"
     value     = "aws-elasticbeanstalk-service-role"
   }
+
+  # Application Deployments
+  setting {
+    namespace = "aws:elasticbeanstalk:command"
+    name      = "DeploymentPolicy"
+    value     = "Immutable"
+  }
+
+  # Configuration Updates
+  setting {
+    namespace = "aws:autoscaling:updatepolicy:rollingupdate"
+    name      = "RollingUpdateEnabled"
+    value     = "true"
+  }
+  setting {
+    namespace = "aws:autoscaling:updatepolicy:rollingupdate"
+    name      = "RollingUpdateType"
+    value     = "Health"
+  }
+  setting {
+    namespace = "aws:autoscaling:updatepolicy:rollingupdate"
+    name      = "MaxBatchSize"
+    value     = "3"
+  }
+  setting {
+    namespace = "aws:autoscaling:updatepolicy:rollingupdate"
+    name      = "MinInstancesInService"
+    value     = "${var.eb_max_size == "1" ? 0 : 1}"
```

Comment:
Should we make this configurable? on production we probably never want to leave only 1 in service?